stdlib/public/Concurrency/CMakeLists.txt

```diff
@@ -29,5 +30,4 @@ add_swift_target_library(swift_Concurrency ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} I
     ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS}
     -parse-stdlib
   LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
-  DARWIN_INSTALL_NAME_DIR "@rpath"
```

Comment:
Note: this change is right -- the install_name should be left with the default /usr/lib/swift/ prefix, matching the other stdlib libraries.